atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepableTimestamps.java

```diff
@@ -73,13 +73,13 @@ private SweepableTimestampsTable.SweepableTimestampsColumn computeColumn(long st
      *
      * @param shardStrategy desired shard and strategy
      * @param lastSweptTs exclusive minimum timestamp to check for
-     * @param sweepTs exclusive maximum timestamp to check for
+     * @param maxTsExclusive exclusive maximum timestamp to check for
      * @return Optional containing the fine partition, or Optional.empty() if there are no more candidates before
      * sweepTs
      */
-    Optional<Long> nextTimestampPartition(ShardAndStrategy shardStrategy, long lastSweptTs, long sweepTs) {
+    Optional<Long> nextTimestampPartition(ShardAndStrategy shardStrategy, long lastSweptTs, long maxTsExclusive) {
         long minFineInclusive = SweepQueueUtils.tsPartitionFine(lastSweptTs + 1);
-        long maxFineInclusive = SweepQueueUtils.tsPartitionFine(sweepTs - 1);
+        long maxFineInclusive = SweepQueueUtils.tsPartitionFine(maxTsExclusive - 1);
```

Comment:
this looks random but was consistent with `SweepableCells` usage, as well as the javadoc comments.